cypress-shared/support/shopper-location/outputvalidation.js

```diff
@@ -31,4 +31,16 @@ export default {
     country: 'United States',
     postalCode: '93206',
   },
+  UsDetails3: {
```

Comment:
This is added in shopper location 
Do we have for location availbility
If yes move or else let it be